### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by swhammack** [mark-hahn/view-tail-large-files#13]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Function.module.exports.FilePickerView.getViewFromDOM (/Users/shammack/.atom/packages/view-tail-large-files/node_modules/vtlf-file-picker/lib/file-picker-view.coffee:82:23)
```

**Comment by mark-hahn** [comment]
When working on this issue I found that view-tail-large-files seems to be very broken on my system. I suspect it might be something to do with a recent Atom version breaking it.  Can you check it for me? 

Are you on the latest Atom version?  Can you do an overall test of view-tail-large-files?  Thanks in advance?

**Comment by swhammack** [comment]
Yes these errors happen on the newest version. I'm running Mac OSX 10.10.12 and Atom 0.187.0. Seems to happen when scrolling quickly.

**Comment by mark-hahn** [comment]
thanks.  Fixed in 0.1.15
